.gitignore

```diff
@@ -24,3 +24,6 @@ coverage
 build/
 dist/bundle.js
 database/generated_data
+database/postgres_data/
+database/config.js
+.env
```

Comment:
I deleted your newrelic.js file from your root directory as it contained your auth key, may want to add newrelic.js to your gitignore